nuxeo-page-provider-display-behavior.html

```diff
@@ -253,6 +253,13 @@
         }
       },
 
+      // ELEMENTS-482: introducing a getter for selected items because the justified grid doesn't rely on iron-list to
+      // to manage selection. In the future, we might want to drop iron-list selection and do the same for other grids
+      // and lists.
+      getSelectedItems: function() {
```

Comment:
I thought we had a `selectedItems` property in the behavior, no? Is this not kept in sync with the current selection?